packages/desktop-electron/package.json

```diff
@@ -17,14 +17,11 @@
     "appId": "com.actualbudget.actual",
     "files": [
       "build",
-      "!node_modules/loot-core/src{,/**/*}",
```

Comment:
No longer need to exclude stuff manually because we're no longer including all of loot-cores dependencies